content/_guides/sasl/emacs-erc.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Configuring SASL for Emacs ERC
+category: sasl
+credits: androclus
+---
+
+[ERC](https://www.emacswiki.org/emacs/ERC) is an IRC client which comes as a package for Emacs.
+
+## A. Set up ERC IRC client
+
+If you have ERC already installed and running on your emacs system, then skip to part B.
+Otherwise, set up your ERC with basic functionality, as follows:
+
+1. Download and Install the erc package from melpa, etc. (*M-x package-list-packages*)
+2. Set up your ERC for plain (unregistered) sign-in.
+  - There are some basic setup instructions [here](https://www.emacswiki.org/emacs/ERC)
+  - Below is my own setup (before adding the SASL stuff below), though by no means do you need all this to set up a basic ERC for yourself.
+```
```

Comment:
Add a blank line before code-block starts, als mark all code blocks with a language. `emacs` or `elisp` is aliases I believe

````suggestion
  - Below is my own setup (before adding the SASL stuff below), though by no means do you need all this to set up a basic ERC for yourself.

```emacs
````

Reply:
Okay, this is done